test/crontab/date_checker_test.exs

```diff
@@ -77,4 +78,27 @@ defmodule Crontab.DateCheckerTest do
       matches_date?(%Crontab.CronExpression{reboot: true}, base_date)
     end
   end
+
+  test "DST ambiguity checks correct" do
+    normal_date = DateTime.from_naive!(~N[2024-11-28 00:00:00], "Europe/Zurich")
+
+    {:ambiguous, date_earlier, date_later} =
+      DateTime.from_naive(~N[2024-10-27 02:30:00], "Europe/Zurich")
+
+    assert matches_date?(~e[* * * * * *]a, date_earlier)
+    refute matches_date?(~e[* * * * * *]a, date_later)
```

Comment:
As the cron expression expects the frequency to be "at every minute", so `shift/4` ignores the ambiguity opt, this test is invalid.